src/MongoDB.Driver.Core/Core/Connections/IsMasterResult.cs

```diff
@@ -382,5 +400,18 @@ public ReplicaSetConfig GetReplicaSetConfig()
 
             return new ReplicaSetConfig(members, name, primary, version);
         }
+        
+        /// <summary>
+        ///     Creates a new IsMasterResult with the SaslSsupportedMechanisms key-value pair merged in.
+        ///     Overwrites existing values.
+        /// </summary>
+        /// <param name="saslSsupportedMechanisms">The SaslSupportedMechanisms to merge in.</param>
+        /// <returns></returns>
+        public IsMasterResult MergeSaslSupportedMechanisms(BsonDocument saslSsupportedMechanisms)
+        {
+            var clone = (BsonDocument) _wrapped.DeepClone();
```

Comment:
Is cloning necessary?

Actually... I don't think this method is even called any more.

Reply:
Yep, no longer used, will remove.